packages/react-native/React/DevSupport/RCTInspectorDevServerHelper.mm

```diff
@@ -78,8 +80,29 @@
   // A bundle ID uniquely identifies a single app throughout the system. [Source: Apple docs]
   NSString *bundleId = [[NSBundle mainBundle] bundleIdentifier];
 
+#if !TARGET_OS_OSX // [macOS]
   // An alphanumeric string that uniquely identifies a device to the app's vendor. [Source: Apple docs]
   NSString *identifierForVendor = [[UIDevice currentDevice] identifierForVendor].UUIDString;
+#else // [macOS
```

Comment:
(nit) missing closing bracket `]`?

Reply:
Nope, the closing bracket is part of the endif. See https://github.com/microsoft/react-native-macos/blob/main/docs/DiffsWithUpstream.md